src/pages/index.js

```diff
@@ -1,47 +1,79 @@
 // IMPORTS
-import { options, initialCards } from "../utils/constants.js";
+import { options } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
-import UserInfo from "../components/UserInfo.js";
+import User from "../components/User.js";
+import Api from "../components/Api.js";
 import "../pages/index.css";
 
 // MODALS
 const editProfileModal = document.querySelector("#edit-modal");
-const addCardModal = document.querySelector("#add-modal");
 const imageModal = new PopupWithImage("#image-modal");
 
 // FORMS
+const avatarForm = document.forms["avatar-form"];
 const profileForm = document.forms["profile-form"];
 const profileFormName = editProfileModal.querySelector("[name = 'name']");
-const profileFormDesc = editProfileModal.querySelector(
-  "[name = 'description']"
-);
+const profileFormDesc = editProfileModal.querySelector("[name = 'about']");
 const addCardForm = document.forms["add-card-form"];
-const userInfo = new UserInfo({
+const user = new User({
   nameSelector: ".profile__name",
-  descriptionSelector: ".profile__description",
+  aboutSelector: ".profile__description",
+  avatarSelector: ".profile__image",
 });
+const popupAvatarForm = new PopupWithForm(
+  "#avatar-modal",
+  avatarFormSubmitHandler
+);
 const popupProfileForm = new PopupWithForm(
   "#edit-modal",
   profileFormSubmitHandler
 );
 const popupAddCardForm = new PopupWithForm(
   "#add-modal",
-  handleAddCardFormSubmit
+  addCardFormSubmitHandler
 );
+const popupDelete = new PopupWithForm("#delete-modal", handleDeleteSubmit);
 
 // FORM VALIDATION
+const avatarFormValidator = new FormValidator(options, avatarForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```